Dockerfile.f28.py3

```diff
@@ -0,0 +1,76 @@
+FROM fedora:28
+ENV LANG C
+ARG type=local
+
+COPY dnf-docker-test/repo /var/www/html/repo/
+COPY dnf-docker-test/features /behave/
+COPY dnf-docker-test/features /tests/
+COPY rpms /rpms/
+COPY dnf-docker-test/x509certgen /usr/local/bin
+
+RUN set -x && \
+    echo -e "deltarpm=0" >> /etc/dnf/dnf.conf && \
+    # httpd:        http-style repos
+    # vsftpd:       ftp-style repos
+    # behave:       core
+    # six:          py2/py3
+    # enum34:       enum for py2, rpmdb.State
+    # whichcraft:   shutil.which() for py2
+    # jinja2:       rpmspec template
+    # pexpect:      shell tests
+    # rpm-build:    building dummy RPMs
+    # openssl:      generating TLS certificates
+    # mod_ssl:      https-style repos
+    # gnupg2:       GPG keys
+    # rng-tools:    to generate enough _random_ data for GPG keys
+    # rpm-sign:     rpm signing
+    # createrepo_c: building repos
+    dnf -y install httpd vsftpd python3-behave python3-six python-enum34 python3-whichcraft python3-jinja2 python3-pexpect rpm-build openssl mod_ssl gnupg2 rng-tools rpm-sign createrepo_c && \
```

Comment:
`python-enum34` is py2

Reply:
Thank you. I have removed it completely as it should be provided by python3 directly.